### PR TITLE
T engg hammero1

### PR DESCRIPTION
**Pull request by TEngg-Hammero1**
Added POM Home_page class
Modified createorg and createorganddelete org test case

### CODE REVIEW

File: src/main/java/com/vtigercrm/objectrepositoryLib/Home_Page.java [hunk 1]
```diff
@@ -1,6 +1,29 @@
...
+	}
+
+	public WebElement getOrglink() {
+		return Orglink;
+	}
 
+	public WebElement getContactlink() {
+		return Contactlink;
+	}
+
```
Comment (Vidush96) [inline src/main/java/com/vtigercrm/objectrepositoryLib/Home_Page.java]: Create business specific method
```suggestion

	public WebElement getOrglink() {
		return Orglink;
	}

	public WebElement getContactlink() {
		return Contactlink;
	}
	

```

**Review by Vidush96 — COMMENTED** [review]
Do changes

File: src/main/java/com/vtigercrm/objectrepositoryLib/Home_Page.java (hunk 1 above)
Comment (TEngg-Hammero1) [reply src/main/java/com/vtigercrm/objectrepositoryLib/Home_Page.java]: k will modify

Comment (TEngg-Hammero1) [reply src/main/java/com/vtigercrm/objectrepositoryLib/Home_Page.java]: have modified code please approve and merge

**Review by Vidush96 — APPROVED** [review]

**Pull request merged**